addons/hr_holidays/models/hr_holidays.py

```diff
@@ -528,7 +528,8 @@ def _prepare_holidays_meeting_values(self):
             'stop': self.date_to,
             'allday': False,
             'state': 'open',  # to block that meeting date in the calendar
-            'privacy': 'confidential'
+            'privacy': 'confidential',
+            'event_tz': self.user_id.tz,
```

Comment:
Is it required since a holiday is not a recurring event?

Reply:
yes, we should add the employees timezone when creating the event. It's done upon validation, which is not done by the employee but by a manager. Unless we make the assumption the manager shares the same timezone it could be wrong.